main.go

```diff
@@ -218,6 +220,25 @@ func main() {
 		setupLog.Info("Skipping setup of Knative Service validating/mutating Webhook, because KServe Serverless setup seems to be disabled in the DataScienceCluster resource.")
 	}
 
+	nimEnabled, nimEnabledErr := utils.VerifyIfComponentIsEnabled(context.Background(), mgr.GetClient(), utils.KServeWithNIMComponent)
```

Comment:
@etirelli @israel-hdez @lburgazzoli @TomerFi @mpaulgreen
I saw the other webhook `ksvcValidatorWebhook` checks the DSC flag before installation.
So I check the nim flag before installing the nim webhook.
Is this expected?

Reply:
same as I mentioned here - https://github.com/opendatahub-io/odh-model-controller/pull/288#discussion_r1832916829

Reply:
Removed the checking for the DSC flag: https://redhat-internal.slack.com/archives/C07CZDA6DE1/p1731018233090329?thread_ts=1730969824.426219&cid=C07CZDA6DE1